x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -43,11 +43,12 @@ import (
 )
 
 const (
-	requestRetrySleepEnv     = "KIBANA_REQUEST_RETRY_SLEEP"
-	maxRequestRetriesEnv     = "KIBANA_REQUEST_RETRY_COUNT"
-	defaultRequestRetrySleep = "1s"                             // sleep 1 sec between retries for HTTP requests
-	defaultMaxRequestRetries = "30"                             // maximum number of retries for HTTP requests
-	defaultStateDirectory    = "/usr/share/elastic-agent/state" // directory that will hold the state data
+	requestRetrySleepEnv          = "KIBANA_REQUEST_RETRY_SLEEP"
+	maxRequestRetriesEnv          = "KIBANA_REQUEST_RETRY_COUNT"
+	defaultRequestRetrySleep      = "1s"                             // sleep 1 sec between retries for HTTP requests
+	defaultMaxRequestRetries      = "30"                             // maximum number of retries for HTTP requests
+	defaultStateDirectory         = "/usr/share/elastic-agent/state" // directory that will hold the state data
+	defaultFleetPackagePolicyName = "default-fleet-server-agent-policy"
```

Comment:
this should be called `fleet-server-policy`

Reply:
@juliaElastic does the default agent policy have an analog to this id? i would like to add another config value for that case as well.

Reply:
no, there is no analog for agent policy

Reply:
it looks like there was a constant used for the policy name before: https://github.com/elastic/kibana/blob/7.17/x-pack/plugins/fleet/common/constants/preconfiguration.ts#L40

would that be safe to use as a default value for the agent policy case?

Reply:
yeah, that is the default policy removed from 8.1. It might make sense for migrated clusters where it still exists.